wger/software/templates/features.html

```diff
@@ -211,18 +197,17 @@ <h4>{% trans "REST API" %}</h4>
                                 <div class="timeline-body">
                                     <p class="text-muted">
                                         {% blocktrans trimmed %}
-                                            wger provides a REST API with full read and write access to all
-                                            data so you can integrate your own application and tools.
+                                            A REST API with full read and write access to all
+                                            data is provided to integrate your own apps and tools.
                                         {% endblocktrans %}
                                     </p>
                                 </div>
                             </div>
                         </li>
                         <li class="timeline">
                             <div class="timeline-image">
-                                <h4>{% blocktrans trimmed %}100% Free
-                                    <br>and
-                                    <br>Open Source{% endblocktrans %}
+                                <h4>{% blocktrans trimmed %}AGPLv3+
```

Comment:
Can you change this to just AGPL3+, it looks nicer when its rendered all in caps